edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/client/KubernetesClient.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.Client
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using k8s;
+    using k8s.Models;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class KubernetesClient
+    {
+        public string DeviceNamespace { get; }
+
+        public IKubernetes Kubernetes { get; }
+
+        public KubernetesClient(string deviceNamespace, IKubernetes client)
+        {
+            this.DeviceNamespace = deviceNamespace;
+            this.Kubernetes = client;
+        }
+
+        public async Task<Option<V1PodList>> WaitUntilAnyPods(string fieldSelector, CancellationToken token) =>
```

Comment:
```suggestion
        public async Task<Option<V1PodList>> WaitUntilAnyPodsAsync(string fieldSelector, CancellationToken token) =>
```

Reply:
Same for other methods.

Reply:
I'll admit I don't like this pattern of decorating the method name with "Async" (an implementation detail that can easily be discovered by looking at the signature). But it's a broadly accepted pattern in C#, and it's the standard in our codebase.

Reply:
👌